### DISCUSSION TITLE
Default configuration, config merging

### DISCUSSION THREAD

**Discussion by octvs** [glzr-io/glazewm#570] [Q&A]
Hey everyone, 
Does GlazeWM employ/able to do some sort of config merging?

As far as I can experiment it doesn't. Commenting out a keybind from my config
doesn't result in inheriting from [sample
config](https://github.com/glzr-io/glazewm/blob/main/GlazeWM.App/Resources/sample-config.yaml).
Since there is no `include` command I can't inherit explicitly.

What I would like is to define a subset of options where I differ from the
defaults, which would result in a more managable config file, probably as a
part of my dotfiles.

Thanks for the replies in advance.

**Comment by MyGitHubBlueberry** [comment]
Maybe you not reloading glaze after making changes? `Ctrl-Shift-R` is default binding for that.

**Comment by octvs** [comment]
> Maybe you not reloading glaze after making changes? 
Could you not reproduce the issue?
